shared/searchv3/results-list/index.desktop.js

```diff
@@ -29,15 +23,8 @@ class SearchResultsList extends Component<void, Props, void> {
 
   render() {
     const {style, items} = this.props
-
-    // TODO maybe move to container so this is shared
```

Comment:
i was thinking you could actually do the switching of empty results in a recompse branch() call and not have these components do the checking at all